Modules/DevTools/BusinessCentralPerformanceToolkit/src/BCPTTestParamProvider.Interface.al

```diff
@@ -1,6 +1,6 @@
 interface "BCPT Test Param. Provider"
 {
-    procedure GetDefaultParameters(): Text[1000];
+    procedure GetDefaultParameters(BCPTLine: Record "BCPT Line"): Text[1000];
```

Comment:
This interface will be implemented by the test codeunit and does not have any relation to the BCPTLine.